examples/rtk_base_ublox.py

```diff
@@ -96,7 +96,7 @@ async def run():
     await drone.connect()
 
     # Start the tasks
-    tasks = [
+    _tasks = [
```

Comment:
It is unusual to assemble these tasks but not await them.  Are we certain that they will be run at some point?